src/Nox.Types/Types/Temperature/TemperatureConversionFactor.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Nox.Types.Common;
+using System.Collections.Generic;
+
+namespace Nox.Types;
+
+public class TemperatureConversionFactor : MeasurementConversionFactor<TemperatureUnit>
+{
+    private static readonly Dictionary<(TemperatureUnit, TemperatureUnit), double> _definedLengthConversionFactors = new()
```

Comment:
Naming starting with underscore is used for private instance fields. 
Static readonly fields starts with capital letter -> `DefinedLengthConversionFactors`

Reply:
It was named after other Cnversions: AreaConversion, WeightConversion, VolumeConversion .. these all are private static readonly fields. Should we change naming for them all, or leave as is since it is a private variable?